src/client/theme-default/composables/sidebar.ts

```diff
@@ -49,6 +49,12 @@ export function useSidebar() {
     )
   })
 
+  const leftAside = computed(() => {
+    if (hasAside && frontmatter.value.aside != null)
+      return frontmatter.value.aside === 'left'
```

Comment:
It should be something like this IG, can you test this locally? (Also run prettier once before committing)

```suggestion
    if (hasAside)
      return frontmatter.value.aside == null ? theme.value.aside === 'left' : frontmatter.value.aside === 'left'
```

CASES:

1. `aside` is set to `left` in `themeConfig`:
   1. if `aside` is not set in frontmatter, then render it on left
   3. if `aside` is set to `true` in frontmatter, then render it on right
   5. if `aside` is set to `false` in frontmatter, then don't render it
2. `aside` is set to `true` or not specified in `themeConfig`:
   1. if `aside` is not set in frontmatter, then render it on right
   2. if `aside` is set to `false` in frontmatter, then don't render it
   3. if `aside` is set to `left` in frontmatter, then render it on left
3. `aside` is set to `false` in `themeConfig`:
   1. if `aside` is not set in frontmatter, then don't render it
   2. if `aside` is set to `left` in frontmatter, then render it on left
   3. if `aside` is set to `true` in frontmatter, then render it on right